src/System.Private.CoreLib/shared/System/Collections/Generic/List.cs

```diff
@@ -23,8 +23,8 @@ public class List<T> : IList<T>, IList, IReadOnlyList<T>
     {
         private const int DefaultCapacity = 4;
 
-        private T[] _items; // Do not rename (binary serialization)
-        private int _size; // Do not rename (binary serialization)
+        internal T[] _items; // Do not rename (binary serialization)
```

Comment:
Is it worth a comment that these fields are internal only for the use of CollectionsMarshal and to avoid further growing the generated code for List? It's not something we normally do.